### ISSUE TITLE
Pause functionality

### ISSUE THREAD

**Issue by sebpettersson** [mvilledieu/shadertoy-react#9]
It might be useful to be able to pause shader. An easy solution might be a boolean prop that if set to `true` makes an early `return` in `drawScene()`.

`shouldComponentUpdate()` would have to be updated to compare this new prop instead of just returning `false`.

**Comment by mvilledieu** [comment]
@sebpettersson True, I'll do that whenever I get some time to do so, otherwise feel free to make a PR :)
